core/server/web/shared/middlewares/update-user-last-seen.js

```diff
@@ -0,0 +1,15 @@
+const constants = require('../../../lib/constants');
+
+module.exports = function updateUserLastSeenMiddleware(req, res, next) {
+    if (!req.user) {
+        return next();
+    }
+
+    if (Date.now() - req.user.get('last_seen') < constants.ONE_HOUR_MS) {
+        return next();
+    }
+
+    req.user.updateLastSeen().then(() => {
```

Comment:
Two thoughts on this:

1. We update the user, but we don't update `req.user` after the successful update. Probably doesn't matter right now, because Ghost does not return `req.user` (only for internal usages)
2. Should we just continue with the request if this operation fails?

Reply:
Ok agree to both :)